ants/viz/plot.py

```diff
@@ -2337,7 +2341,7 @@ def reorient_slice(x, axis):
             # resample image if spacing is very unbalanced
             spacing = [s for i, s in enumerate(image.spacing) if i != axis]
             was_resampled = False
-            if (max(spacing) / min(spacing)) > 3.0:
+            if (max(spacing) / min(spacing)) > 3.0 and resample:
                 was_resampled = True
                 new_spacing = (1, 1, 1)
```

Comment:
this line should be (pseudo - code here ):

repeat min(spacing) image-dimensions times

Reply:
Don't understand your comment.